Console_Arraylist/MyArrayList.cs

```diff
@@ -16,17 +16,22 @@ public class MyArrayList : IList
         private const int _defaultCapacity = 4;
         private object[] emptyarray = new object[0];
 
-        public MyArrayList(int capacity)
+        public MyArrayList()
```

Comment:
Why you replaced constructor with empty one, It's better to have both constructors one with capacity, the other one without capacity. Please refactor code to have both constructors.